### PR TITLE
document things that are useful to know for using tinkr

### LINKED ISSUE #36: document things that are useful to know for using tinkr
* XPath https://www.w3schools.com/xml/xpath_intro.asp

* XSLT (not crucial if they are ok with the default one) https://ropensci.org/blog/2017/01/10/xslt-release/

* R6 (although reading the docs of tinkr should help)

* how xml2 works https://blog.r-hub.io/2020/01/22/mutable-api/#exposing-the-c-api-in-xml2

### PR DESCRIPTION
**Pull request by maelle**
Fix #36

### CODE REVIEW

**Comment by zkamvar** [comment]
I promise I will look at this tomorrow!

**Comment by maelle** [comment]
no pressure, no hurry!

File: README.Rmd [hunk 1]
```diff
@@ -132,6 +133,14 @@ rmd$add_md(knitr::kable(new_table), where = 2L)
...
 ```
 
+## Background / pre-requisites
+
+If you are not closely following one of the examples provided, what background knowledge do you need before using tinkr?
+
+* That XPath, a language for querying XML & HTML, exists, and [some basics](https://www.w3schools.com/xml/xpath_intro.asp).
+* Basics of how [xml2 works](https://blog.r-hub.io/2020/01/22/mutable-api/#exposing-the-c-api-in-xml2): how to find, replace, remove nodes etc.
+* How to use R6 classes... although reading the examples should help you get the gist.
+* [XSLT](https://ropensci.org/blog/2017/01/10/xslt-release/) but only if you are not happy with our [default stylesheet](#general-principles-and-solution).
```
Comment (zkamvar) [inline README.Rmd:143]: I think we should state the caveat first, or else we will lose people who thing the ALSO need to know XSLT. 

```suggestion
* If you are not happy with [our default stylesheet](#general-principles-and-solution), then understanding [XSLT](https://ropensci.org/blog/2017/01/10/xslt-release/) will help you create your own.
```

File: README.Rmd [hunk 2]
```diff
@@ -132,6 +133,14 @@ rmd$add_md(knitr::kable(new_table), where = 2L)
 rmd$head(21)
 ```
 
+## Background / pre-requisites
```
Comment (zkamvar) [inline README.Rmd:136]: I think this needs to be linked on line 26 so that people who are completely lost will know where to go.

**Review by zkamvar — CHANGES_REQUESTED** [review]
Looks good to me! I only have a couple of suggestions about order of ideas and links.

**Pull request merged**